### PR TITLE
Magento 123

### PR DESCRIPTION
**Pull request by suraiyavic**

### CODE REVIEW

File: etc/adminhtml/system.xml [hunk 1]
```diff
@@ -183,6 +183,13 @@
                         <validate>required-entry validate-length minimum-length-2 maximum-length-20 validate-no-empty reach-default-hs-code-validator</validate>
                         <config_path>reach/dhl/default_hs_code</config_path>
                     </field>
+                    <!-- Default Country of Origin of a Product - related to Default HS Code -->
+                    <field id="default_country_of_origin" translate="label" type="multiselect" sortOrder="115" showInDefault="1" showInWebsite="1" showInStore="0">
```
Comment (lizzyWithReach) [inline etc/adminhtml/system.xml]: I would have expected the default country of origin to have the same visibility as default_hs_code, but showInWebsite is 0 for default country of origin but 1 for default_hs_code. Wanted to check that that's ok

**Review by lizzyWithReach — CHANGES_REQUESTED** [review]
Just want to check on the visibility of the default country of origin

File: etc/adminhtml/system.xml (hunk 1 above)
Comment (lizzyWithReach) [reply etc/adminhtml/system.xml]: sorry I meant "showInStore", not showInWebsite

Comment (suraiyavic) [reply etc/adminhtml/system.xml]: commit 2f694bcc469db49bf3846ec11d7005abb7edb0ac
Thanks!

**Review by lizzyWithReach — APPROVED** [review]
Thank you!

**Pull request merged**